First draft documentation

sudhirunicus opened the pull request:
Please provide feedback and information for these documents

sudhirunicus committed:
First draft

sudhirunicus committed:
Delete unnecesarry files

Unicus-Tom commented on docs/platform/SEPES component overview.md:
```diff
@@ -0,0 +1,121 @@
+
+## SEPES Frontend ##
+
+Frontend allowes users to manage 
```
Incomplete sentence

Unicus-Tom commented on docs/platform/SEPES component overview.md:
```diff
@@ -0,0 +1,121 @@
+## Azure Portal ##
+
+Azure Portal is used by vendors to create resources for their experiment. 
+Documentation can be found at microsoft homepage
```
https://docs.microsoft.com/nb-no/azure/

gjersvik replied:
https://docs.microsoft.com/azure/ Should use in international one that redirects to local version.

Unicus-Tom commented on docs/platform/SEPES component overview.md:
```diff
@@ -0,0 +1,121 @@
+
+Coordinate the different services, and make actions on the behalf of the user.
+
+Tech Stack: asp.net core, C#
```
dotnet, asp.net, C#

gjersvik replied:
We need the core part. .net and ASP.net is not the same as .net core and ASP.net core. At least until .net 5 comes out in a few years. Also think we should add the azure sdk as part of the tech stack.

Unicus-Tom commented on docs/platform/SEPES component overview.md:
```diff
@@ -0,0 +1,121 @@
+
+
+
+Maturity is based on Maufacturing Readiness Level. https://rescoll.fr/wp-content/uploads/2014/01/MRL.jpg
```
These should be in same order as is done in other uses of maturity and protocol

gjersvik replied:
I think we should look for another more appropriate maturity metrics. To use. After closer look Maufacturing Readiness Level do not seem appropriate.

Unicus-Tom commented on docs/platform/SEPES component overview.md:
```diff
@@ -0,0 +1,121 @@
+
+Maturity: 
+
+Link for Azure AD use. https://docs.microsoft.com/en-us/azure/active-directory/
```
Should be moved up to describtion

gjersvik commented on docs/platform/SEPES component overview.md:
```diff
@@ -0,0 +1,121 @@
+- Sponsors: Responsible for sandbox management 
+- Supplier: External vendors
+
+Tech Stack: 
```
- React JS https://reactjs.org/
- WebPack https://webpack.js.org/
- HTML, CSS and JavaScript

sudhirunicus committed:
second draft

The following changes for document are made according to comments. Please question or comment more.

gjersvik commented on docs/platform/SEPES component overview.md:
```diff
@@ -0,0 +1,121 @@
+
+Protocol: Uses https rest protocol and can be configured to use http. http should only be used for cases that requires it.
+
+As an example docker compose is configured to use http, this is still safe as the docker wrapper uses https outside.
```
Its only safe its behind a proxy that handle HTTPS terminations. Or in local development setting. Docker compose have nothing to do with protocol on it own. Proxy is the important keyword.

gjersvik reviewed (approved):
Good first draft but there are tings that need to be looked at for a next version.

gjersvik commented on docs/platform/SEPES component overview.md:
```diff
@@ -4,7 +4,7 @@ In this document SEPES, with its different components, will be described/ docume
 ## SEPES Frontend ##
 
-Frontend allowes users to manage 
+Frontend allows users to manage pods.
```
Think we should use sandboxes and not pods.

gjersvik commented on docs/platform/SEPES component overview.md:
```diff
@@ -32,7 +32,7 @@ Protocol: https
 Coordinate the different services, and make actions on the behalf of the user.
 
-Tech Stack: asp.net core, C#
+Tech Stack: dot.net, asp.net core, C#
```
Its ".net" not dotnet or dot.net

gjersvik commented on docs/platform/SEPES component overview.md:
```diff
@@ -92,11 +92,11 @@ Protocol: Uses http, so have to be driven in the docker. Link documentation at O
 ## Azure AD ##
 
-Used for autentication
+Used for autentication. Link for Azure AD use. https://docs.microsoft.com/en-us/azure/active-directory/
```
https://docs.microsoft.com/azure/active-directory/ International link

sudhirunicus committed:
changes 2. draft

Changes made according to comments

gjersvik commented on docs/platform/SEPES component overview.md:
```diff
@@ -32,11 +35,13 @@ Protocol: https
 Coordinate the different services, and make actions on the behalf of the user.
 
-Tech Stack: dot.net, asp.net core, C#
+Tech Stack: .net, asp.net core, C#
```
Tech Stack: .net core, asp.net core, C#

gjersvik commented on docs/platform/SEPES component overview.md:
```diff
@@ -32,11 +35,13 @@ Protocol: https
-Protocol: Uses https rest protocol and can be configured to use http. http should only be used for cases that requires it.
+Protocol: Uses https rest protocol and can be configured to use http. http should only be used for cases that requires it. 
+
+Its only safe its behind a proxy that handle HTTPS terminations. Or in local development setting. Docker compose have nothing to do with protocol on it own. Proxy is the important keyword.
```
Its only safe its behind a proxy that handle HTTPS terminations. Or in local development setting. ~~Docker compose have nothing to do with protocol on it own. Proxy is the important keyword.~~

gjersvik commented on docs/platform/SEPES component overview.md:
```diff
@@ -92,8 +97,7 @@ Protocol: Uses http, so have to be driven in the docker. Link documentation at O
 
-Used for autentication. Link for Azure AD use. https://docs.microsoft.com/en-us/azure/active-directory/
-
+Used for autentication. Link for Azure AD use. https://docs.microsoft.com/nb-no/azure/active-directory/
```
Used for autentication. Link for Azure AD use. https://docs.microsoft.com/azure/active-directory/

sudhirunicus committed:
Further changes made

sudhirunicus commented:
new changes made

sudhirunicus reviewed (commented):
check new changes

kribjo1968 reviewed (approved):
Looks good with OMG's corrections.
